docs/known-qualifiers.md

```diff
@@ -5,40 +5,41 @@ keys but their usage should be limited to the bare minimum for proper package
 identification to ensure that a PURL  stays compact and readable in most
 cases.
 
-Additional, separate external attributes stored outside of a PURL  are the
+Additional, separate external attributes stored outside of a PURL are the
 preferred mechanism to convey extra long and optional information such as a
 download URL, VCS URL or checksums in an API, database or web form.
 
-With this warning, the known **key** and **value**` defined here are valid for
+With this warning, the known **key** and **value** defined here are valid for
 use in all package types:
 
 - VERS allows the specification of a version range.
-  The value must adhere to the `Version Range Specification`.
-  This qualifier is mutually exclusive with the `version` component.
+  The value must adhere to the [Version Range Specification](https://github.com/package-url/vers-spec/blob/main/VERSION-RANGE-SPEC.md).
+  This qualifier is mutually exclusive with the **version** component.
   For example:
 
-    pkg:pypi/django?vers=vers:pypi%2F%3E%3D1.11.0%7C%21%3D1.11.1%7C%3C2.0.0
+      pkg:pypi/django?vers=vers:pypi%2F%3E%3D1.11.0%7C%21%3D1.11.1%7C%3C2.0.0
 
-- `repository_url` is an extra URL for an alternative, non-default package
+- 'repository_url' is an extra URL for an alternative, non-default package
   repository or registry. When a package does not come from the default
-  public package repository for its **type** a PURL  may be qualified with
+  public package repository for its **type** a PURL may be qualified with
   this extra URL. The default repository or registry of a **type** is
-  documented in the "Registered PURL  types" section.
+  documented in the "Registered PURL types" section.
 
-- `download_url` is an extra URL for a direct package web download URL to
+- 'download_url' is an extra URL for a direct package web download URL to
   optionally qualify a PURL .
 
-- `vcs_url` is an extra URL for a package version control system URL to
-  optionally qualify a PURL . The syntax for this URL should be as defined
+- 'vcs_url' is an extra URL for a package version control system URL to
+  optionally qualify a PURL. The syntax for this URL should be as defined
   in Python pip or the SPDX specification. See
   https://github.com/spdx/spdx-spec/blob/cfa1b9d08903/chapters/3-package-information.md#3.7
 
-- `file_name` is an extra file name of a package archive.
+- 'file_name' is an extra file name of a package archive.
 
-- **checksum** is a qualifier for one or more checksums stored as a
-  comma-separated list. Each item in the **value** is in form of
+- 'checksum' is a qualifier for one or more checksums stored as a
+  comma-separated list. Each item in the value is in form of
```

Comment:
Here, too, should "value" be in double asterisks/bold?

Reply:
yes - updated